### PR TITLE
made a blueprint for reviews

### PR DESCRIPTION
**Pull request by nimaester**
current preview of PR

Screen Shot 2020-08-09 at 6 23 26 PM

### CODE REVIEW

File: client/src/Components/App.jsx [hunk 1]
```diff
@@ -1,10 +1,26 @@
 import React from 'react';
+import Stars from './Stars.jsx';
+import RatingsContainer from './RatingsContainer.jsx';
+import Reviews from './Reviews.jsx';
 
 const App = () => {
 
+
+  let [users, getUsers] = React.useState([]);
```
Comment (jgarcia260) [inline client/src/Components/App.jsx:9]: hooks! very nice

**Review by jgarcia260 — APPROVED** [review]
great job. would you be able to implement a high order function like map to reviews.jsx and ratingsContainer.jsx?

File: client/src/Components/App.jsx (hunk 1 above)
Comment (nimaester) [reply client/src/Components/App.jsx:9]: that's the plan. just made the components where data from dB will go. then pass em along as props

Comment (nimaester) [reply client/src/Components/App.jsx:9]: I've been learning hooks. for a couple of days now. hopefully I get it righ

**Pull request merged**